Sources/MAVSDK-Swift/Generated/Passthrough.swift

```diff
@@ -0,0 +1,679 @@
+                && lhs.magic == rhs.magic
+                && lhs.len == rhs.len
+                && lhs.incompatFlags == rhs.incompatFlags
+                && lhs.compatFlags == rhs.compatFlags
+                && lhs.seq == rhs.seq
+                && lhs.sysid == rhs.sysid
+                && lhs.compid == rhs.compid
+                && lhs.msgid == rhs.msgid
+                && lhs.payload64 == rhs.payload64
+                && lhs.ck == rhs.ck
+        }
+    }
+
+    public struct CommandInt: Equatable {
+        public let targetSystemID: UInt32
+        public let targetComponentID: UInt32
+        public let command: UInt64
+        public let current: Bool
+        public let autocontinue: Bool
+        public let params: ParamsInt
+
+        
+
+        public init(targetSystemID: UInt32, targetComponentID: UInt32, command: UInt64, current: Bool, autocontinue: Bool, params: ParamsInt) {
+            self.targetSystemID = targetSystemID
+            self.targetComponentID = targetComponentID
+            self.command = command
+            self.current = current
+            self.autocontinue = autocontinue
+            self.params = params
+        }
+
+        internal var rpcCommandInt: Mavsdk_Rpc_Passthrough_CommandInt {
+            var rpcCommandInt = Mavsdk_Rpc_Passthrough_CommandInt()
+            
+                
+            rpcCommandInt.targetSystemID = targetSystemID
+                
+            
+            
+                
+            rpcCommandInt.targetComponentID = targetComponentID
+                
+            
+            
+                
+            rpcCommandInt.command = command
+                
+            
+            
+                
+            rpcCommandInt.current = current
+                
+            
+            
+                
+            rpcCommandInt.autocontinue = autocontinue
+                
+            
+            
+                
+            rpcCommandInt.params = params.rpcParamsInt
+                
+            
+
+            return rpcCommandInt
+        }
+
+        internal static func translateFromRpc(_ rpcCommandInt: Mavsdk_Rpc_Passthrough_CommandInt) -> CommandInt {
+            return CommandInt(targetSystemID: rpcCommandInt.targetSystemID, targetComponentID: rpcCommandInt.targetComponentID, command: rpcCommandInt.command, current: rpcCommandInt.current, autocontinue: rpcCommandInt.autocontinue, params: ParamsInt.translateFromRpc(rpcCommandInt.params))
+        }
+
+        public static func == (lhs: CommandInt, rhs: CommandInt) -> Bool {
+            return lhs.targetSystemID == rhs.targetSystemID
+                && lhs.targetComponentID == rhs.targetComponentID
+                && lhs.command == rhs.command
+                && lhs.current == rhs.current
+                && lhs.autocontinue == rhs.autocontinue
+                && lhs.params == rhs.params
+        }
+    }
+
+    public struct ParamsInt: Equatable {
+        public let param1: Float
```

Comment:
@douglaswsilva Is it normal that param# is float and x,y is Int and z is Float again?

Reply:
Yes, great question, this is the protocol for CommandInt. :)